charts/gardener/controlplane/charts/application/templates/validatingwebhook-admission-controller.yaml

```diff
@@ -238,7 +238,7 @@ webhooks:
       gardener.cloud/role: project
   clientConfig:
     {{- if .Values.global.deployment.virtualGarden.enabled }}
-    url: https://gardener-admission-controller.garden/webhooks/audit-policies
+    url: https://gardener-admission-controller.garden/webhooks/validate-admission-plugin-secret
```

Comment:
Should this not stay `audit-policies`?

Reply:
Updated in the wrong place. Fixed in 2b053c11bd4dd5999dbda84b312e70bd3a197594